### PR TITLE
Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**
UIListView.h

### CODE REVIEW

**Comment by VisualSJ** [comment]
@natural-law Please review it. Thanks

File: cocos/ui/UIListView.h [hunk 1]
```diff
@@ -47,25 +50,37 @@ typedef enum
...
- *The list items are inserted to the list by using `addChild` or  `insertDefaultItem`.
- * @warning The list item in ListView doesn't support cell reuse at the moment, if you have a large amount of data need to be displayed, use  `TableView` instead.
+ * @brief @~english ListView is a view group that displays a list of scrollable items.
+ * The list items are inserted to the list by using `addChild` or  `insertDefaultItem`.
+ * @~chinese ListView控件是一个显示滚动项目列表的视图组。
+ * 列表项是通过使用`addChild`或`insertDefaultItem`插入到列表中的。
+ * @warning @~english The list item in ListView doesn't support cell reuse at the moment, if you have a large amount of data need to be displayed, use  `TableView` instead.
  * ListView is a subclass of  `ScrollView`, so it shares many features of ScrollView.
+ * @~chinese 在ListView控件列表项不支持复用节点，如果你有大量的数据需要显示，请使用`TableView`代替。
+ * ListView控件是ScrollView的子类，所以它的共享了ScrollView的特点。
```
Comment (natural-law) [inline cocos/ui/UIListView.h]: “的共享”感觉应该是“继承”

File: cocos/ui/UIListView.h [hunk 2]
```diff
@@ -87,125 +105,194 @@ class CC_GUI_DLL ListView : public ScrollView
...
+     * @~chinese Widget的模型
      */
     void setItemModel(Widget* model);
     
     /**
-     * Insert a  a default item(create by a cloned model) at the end of the listview.
+     * @~english
+     * Insert a default item(create by a cloned model) at the end of the listview.
+     * @~chinese 
+     * 插入一个默认项（通过克隆模式创建）的ListView的结尾。
```
Comment (natural-law) [inline cocos/ui/UIListView.h]: 的ListView
到 ListView

**Comment by natural-law** [comment]
@andyque 
It's ok for merge.

**Pull request merged**